File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/IncorrectJwtSubjectException.java
package com.dogGetDrunk.meetjyou.common.exception.business;

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode;

public class IncorrectJwtSubjectException extends CustomJwtException {

    private String value;

    public IncorrectJwtSubjectException(String value) {
        super(value, ErrorCode.INCORRECT_TOKEN_SUBJECT);
        this.value = value;
    }
}

File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/IncorrectJwtSubjectException.kt
package com.dogGetDrunk.meetjyou.common.exception.business

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode

class IncorrectJwtSubjectException(
    value: String
) : CustomJwtException(value, ErrorCode.INCORRECT_TOKEN_SUBJECT) {
    constructor(value: Long) : this(value.toString())
}
